components/map/index.js

```diff
@@ -1,19 +1,34 @@
+/* eslint-disable react/no-unused-state */
```

Comment:
I have noticed quite a few of these inline disable properties for ESLint. It is starting to look messy in teh code. I would suggest to instead disable this in the ESLint setting file

Reply:
Agree, sorry we will fix that asap